### ISSUE TITLE
Wiki markdown file H3's should (almost) always have a <hr/> prior to it

### ISSUE THREAD

**Issue by gonX** [OpenTabletDriver/opentabletdriver.github.io#11]
Correct:

```
### First Title

Bla bla

---

### Second Title

More bla bla
```

Incorrect:
``` 
### First Title

Bla bla

### Second Title

More bla bla
```

It could be accomplished using something like [markdownlint](https://github.com/markdownlint/markdownlint) and connecting it to a GitHub actions similar to [Lint Action](https://github.com/marketplace/actions/lint-action), which does not support Markdown.

/edit:
There are _probably_ situations where we would want to not have the H3, e.g. if the page is short enough. It would be nice if we could opt-out per-paragraph (or at the very least per-page) with some sort of front-matter or HTML attribute.

**Comment by ankman007** [comment]
@gonX Working on this. Will make PR shortly.

**Comment by gonX** [comment]
Implicitly fixed with #63, not as linting but rather as styling.
